airflow/providers/datadog/hooks/datadog.py

```diff
@@ -152,3 +157,27 @@ def post_event(
 
         self.validate_response(response)
         return response
+
+    @staticmethod
+    def get_connection_form_widgets() -> dict[str, Any]:
+        """Returns connection widgets to add to connection form"""
+        from flask_appbuilder.fieldwidgets import BS3TextFieldWidget
+        from flask_babel import lazy_gettext
+        from wtforms import StringField
+
+        return {
+            "api_host": StringField(lazy_gettext("Datadog API endpoint"), widget=BS3TextFieldWidget()),
+            "api_key": StringField(lazy_gettext("Datadog API key"), widget=BS3TextFieldWidget()),
+            "app_key": StringField(lazy_gettext("Datadog application key"), widget=BS3TextFieldWidget()),
+            "source_type_name": StringField(
+                lazy_gettext("Datadog source type name"), widget=BS3TextFieldWidget()
```

Comment:
```suggestion
            "api_host": StringField(lazy_gettext("API endpoint"), widget=BS3TextFieldWidget()),
            "api_key": StringField(lazy_gettext("API key"), widget=BS3TextFieldWidget()),
            "app_key": StringField(lazy_gettext("Application key"), widget=BS3TextFieldWidget()),
            "source_type_name": StringField(
                lazy_gettext("Source type name"), widget=BS3TextFieldWidget()
```
WDYT about removing "Datadog" from the labels? This feels a little redundant given users would have chosen the Datadog conn type and the fields are assumed to be Datadog related.

Reply:
I had the same impression :+1: